pkg/proxy/http.go

```diff
@@ -48,6 +59,7 @@ func (p *Proxy) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	} else {
 		origDestIp = origDest
 	}
+	log.Infof("received request, origDestIp: %s, origDestPort: %d", origDestIp, origDestPort)
```

Comment:
test log? seems to not have full meaning

Reply:
yeah, remove now